lib/quartz/go_process.rb

```diff
@@ -6,8 +6,12 @@ def self.processes
     @processes ||= []
   end
 
+  def seed
+    @seed ||= rand(10000)
```

Comment:
I suspect that the `||=` operator isn't atomic, so maybe we should set `@seed` in the initializer?